frontend/lib/laletterbuilder/site.tsx

```diff
@@ -87,6 +128,23 @@ const LaLetterBuilderSite = React.forwardRef<HTMLDivElement, AppSiteProps>(
               : "jf-norent-internal-above-footer-content"
           )}
         >
+          <div className="jf-laletterbuilder-second-nav is-hidden-tablet">
```

Comment:
So the second nav is the top nav with language and signin? Maybe call it something more descriptive than second-nav, like `language-nav` or `profile-nav` or `top-nav`

Reply:
yes! good point. Changing now.